apps/demo/app/root.tsx

```diff
@@ -14,13 +14,14 @@ export const meta: MetaFunction = () => ({
   viewport: 'width=device-width,initial-scale=1',
 });
 
-import styles from './styles/global.css';
+// import styles from './styles/global.css';
+import theme from "@rangle/radius-react-core-foundations/generated/theme.css";
```

Comment:
Not sure what is a better way to import the theme.css from foundations - I'm getting the module not found error even though the demo app is consuming foundations. Using ts path alias for now to make the theme CSS works.